contentcuration/contentcuration/static/js/edit_channel/new_channel/views.js

```diff
@@ -6,6 +6,7 @@ require("dropzone/dist/dropzone.css");
 var Models = require("edit_channel/models");
```

Comment:
Updated default behavior to open channel details rather than jump straight to channel edit page